src/ZfcDatagrid/Renderer/BootstrapTable/View/Helper/TableRowFactory.php

```diff
@@ -1,21 +1,22 @@
 <?php
 namespace ZfcDatagrid\Renderer\BootstrapTable\View\Helper;
 
+use Zend\ServiceManager\AbstractPluginManager;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class TableRowFactory implements FactoryInterface
 {
     /**
-     * @param  ServiceLocatorInterface $serviceLocator
+     * @param  ServiceLocatorInterface|AbstractPluginManager $serviceLocator
```

Comment:
the method signature is from https://github.com/zendframework/zend-servicemanager/blob/release-2.7.5/src/FactoryInterface.php#L20

the $serviceLocator is `Zend\View\HelperPluginManager` and if we call there, only the get method, than we get only view_plugins and not the "global" services

http://robertbasic.com/blog/working-with-custom-view-helpers-in-zend-framework-2/